backend/benefit/applications/migrations/0013_application_number_auto_field.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.4 on 2021-08-11 16:21
+
+import applications.models
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("applications", "0012_create_application_number_seq"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="application",
+            name="application_number",
+            field=models.IntegerField(
+                default=applications.models._get_next_application_number,
+                verbose_name="application number",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="historicalapplication",
+            name="application_number",
+            field=models.IntegerField(
+                default=applications.models._get_next_application_number,
```

Comment:
Hmn, now when I look at it, does it mean the historical_application will also use the same sequence? At least from the test it doesn't seem so

Reply:
Well, I think Historical will never use the sequence, as it'll always use the value from the origin application, which means it never uses the default value from the model (i.e getting next value from the sequence)

Reply:
ok good to check